jekyll/_cci2/deploy-to-amazon-sagemaker.adoc

```diff
@@ -0,0 +1,368 @@
+
+[#prerequisites]
+== Prerequisites
+
+To follow along with this how-to guide you will need the following:
+
+* A CircleCI account connected to your code. You can link:https://circleci.com/signup/[sign up for free]
+* Basic knowledge of Amazon SageMaker
+* An AWS account with access to SageMaker and a SageMaker studio domain
+* A copy of the link:https://github.com/CircleCI-Public/sagemaker-deploy-examples[example repository] building on CircleCI. For steps to get the project set up, see the xref:create-project#[Create a project in CircleCI] page.
+
+[#set-up-a-circleci-release-environment]
+== 1. Set up a CircleCI release environment
+
+First, set up a release environment to monitor your releases from the CircleCI web app, and access options to manage promotions and rollbacks.
+
+. In the CircleCI web app, select **Releases** from the sidebar
+. If this is your first release environment, you will see an option to btn:[Add Release Environment]. If not, Click btn:[Configure Environments] in the top right corner, and then click btn:[Configure New Environment]
++
+image::deploy/add-release-environment.png[Screenshot showing option to add a new release environment]
+. In the popup, select **Amazon SageMaker** from the "Environment" dropdown. Give your new environment a name, and click btn:[Create Environment]
++
+image::deploy/create-new-environment.png[Screenshot showing new release environment popup]
+. Click on your new environment name to enter the environment view
+. Click btn:[Create New Token]
++
+image::deploy/release-create-new-token.png[Screenshot showing create new token button]
+. Click btn:[Create Token]. Be sure to save this token, you will create an environment variable later to store it.
+
+[#set-up-amazon-sagemaker-access]
+== 2. Set up Amazon SageMaker access
+
+[#oidc-identity-provider]
+=== a. OIDC Identity Provider
+
+In this section you will create an IAM (Identity and Access Management) Identity Provider in your AWS IAM for the CircleCI OIDC Provider. The Amazon SageMaker orb uses OIDC. You may already have this set up, in which case you can skip this section. For more information on CircleCI OIDC functionality, see the xref:openid-connect-tokens#authenticate-jobs-with-cloud-providers[Using OpenID Connect tokens in jobs] page.
+
+. Navigate to your AWS Management Console
+. Go to menu:IAM[Access management > Identity providers]
+. Select **Add Provider**
++
+image::deploy/aws-iam-add-provider.png[Screenshot showing AWS management console Add Provider option]
+. Under "Provider Type", select **OpenID Connect**
+. Enter your "Provider URL". This will be `\https://oidc.circleci.com/org/<your-organization-id>`, where `<your-organization-id>` is the ID of your CircleCI organization.
++
+{% include snippets/find-organization-id.adoc %}
+. Click **Get Thumbprint**
+. For "Audience" enter your organization ID
+. Click btn:[Add Provider]
+
+Next, create a Policy and then a Role.
+
+[#policy]
+=== b. Policy
```

Comment:
I added this section, it wasn't mentioned in the blog draft, so needs checking

Reply:
(leaving for @meeech )